posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -360,6 +361,25 @@ async def initialize_and_resume_multipart_upload(inputs: S3InsertInputs) -> tupl
     return s3_upload, interval_start
 
 
+def s3_default_fields() -> list[BatchExportField]:
+    """Default fields for an S3 batch export.
+
+    Starting from the common defualt fields, we add and tweak some fields for
+    backwards compatibility.
+    """
+    batch_export_fields = default_fields()
+    batch_export_fields.append({"expression": "elements_chain", "alias": "elements_chain"})
+    batch_export_fields.append({"expression": "nullIf(person_properties, '')", "alias": "person_properties"})
+    batch_export_fields.append({"expression": "toString(person_id)", "alias": "person_id"})
+    # In contrast to other destinations, we do export this field.
```

Comment:
I saw this in the other PR, but why is that, btw? Just because we did once and people may expect it?

Reply:
Default schemas for all destinations were originally based on the schemas used in each of the PostHog export plugins. As we were migrating users to batch exports from plugins, we wanted to maintain backwards compatibility with the schemas used in plugins. I do not know why schemas were defined in one or another way in plugins originally though...

For S3 in particular, the schema comes from the [plugin-scaffold repo](https://github.com/PostHog/plugin-scaffold/blob/ff9e30822d119440e911d549aa94f42d2e8c9361/src/types.ts#L236) (as the `ProcessedPluginEvent` is just `JSON.stringify`-ed into the body).

Not sure why we added `inserted_at` on top of that, maybe because blob storage is not as strict with schemas :shrug: Anyways, now that it has been included, again we are just maintaining backwards compatibility.